code/modules/mob/living/silicon/pai/emote.dm

```diff
@@ -0,0 +1,163 @@
+/mob/living/silicon/pai/emote(var/act,var/m_type=1,var/message = null, var/auto)
+	if(timestopped)
+		return //under effects of time magick
+	var/param = null
+	if (findtext(act, "-", 1, null))
```

Comment:
I've never touched regex before and none of the existing `emote()` procs use it.